.github/include/ci.py

```diff
@@ -219,6 +220,43 @@ def tests_finish(results: List[TestResult]):
         print(output.getvalue())
 
 
+def run_runtime_tests():
+    """Runs runtime tests, under a controlled kernel if requested"""
+    script = "./tests/runtime-tests.sh"
+
+    if NIX_TARGET_KERNEL:
+        # Bring kernel into nix store then grab the path
+        subprocess.run([nix(), "build", NIX_TARGET_KERNEL], check=True)
+        eval = subprocess.run(
+            [nix(), "eval", "--raw", NIX_TARGET_KERNEL],
+            check=True,
+            capture_output=True,
+            text=True,
+        )
+
+        # nf_tables and xfs are necessary for testing kernel modules BTF support
+        modules = ["kvm", "nf_tables", "xfs"]
```

Comment:
Would be nice to somehow unify the list of necessary modules to a single place so that we don't have to maintain two lists (one here and one in `ci.yml`).

Reply:
Right. I'm still thinking on that a bit. My feeling is these modules tests are reaching kinda deep into the kernel. So maybe it'd be a good idea to only test under vmtest. Easier to develop / reproduce locally, as it's not easy to run a GHA runner kernel locally.

If we only run module tests under vmtest, then we can drop the modprobes in ci.yml. Lemme follow up on that when we solidify a vmtest-ing policy.

Reply:
Right, that makes sense b/c we can't be sure the modules are compiled in the CI runner kernel. To me, these tests are kernel-dependent and therefore it makes sense to run them under vmtest only.